pkg/server/singleprocess/service_entrypoint.go

```diff
@@ -26,14 +26,20 @@ import (
 	"github.com/hashicorp/waypoint/pkg/serverstate"
 )
 
-// TODO: test
 func (s *Service) EntrypointConfig(
 	req *pb.EntrypointConfigRequest,
 	srv pb.Waypoint_EntrypointConfigServer,
 ) error {
 	log := hclog.FromContext(srv.Context())
 	ctx := srv.Context()
 
+	// Validate CEB token is valid for this deployment
+	if tok := s.decodedTokenFromContext(ctx); tok != nil {
+		if tok.UnusedEntrypoint != nil && tok.UnusedEntrypoint.DeploymentId != req.DeploymentId {
```

Comment:
I believe that the `UnusedEntrypoint` field isn't being populated, so I don't think this will ever fire.

Reply:
Oh lol, you're totally right it is. Maybe update the comment over there that it's being used for authz.